GliaWidgets/Sources/View/Chat/ChatView.swift

```diff
@@ -928,4 +933,69 @@ extension ChatView {
         view.setOperatorImage(fromUrl: imageUrl, animated: false)
         return .gvaPersistentButton(view)
     }
+
+    private func gvaGalleryListViewContent(
+        _ message: ChatMessage,
+        gallery: GvaGallery,
+        showsImage: Bool,
+        imageUrl: String?
+    ) -> ChatItemCell.Content {
+        let listViewEnvironment = GvaGalleryListView.Environment(
+            data: environment.data,
+            uuid: environment.uuid,
+            gcd: environment.gcd,
+            imageViewCache: environment.imageViewCache,
+            uiScreen: environment.uiScreen
+        )
+        let view = GvaGalleryListView(environment: listViewEnvironment)
+        let items: [GvaGalleryCardView.Props] = gallery.galleryCards.map { card in
```

Comment:
NIT: I think this can also be moved to its own method. The retrieval of items makes a lot of noise here.

Reply:
Fixed 4285cdde0a5c765ee09b36d30c2a36d0bfc1b777